src/shapes/path.class.js

```diff
@@ -354,36 +205,6 @@
             y = tempY;
             break;
 
-          case 'a':
-            // TODO: optimize this
-            drawArc(ctx, x + l, y + t, [
-              current[1],
-              current[2],
-              current[3],
-              current[4],
-              current[5],
-              current[6] + x + l,
-              current[7] + y + t
-            ]);
-            x += current[6];
-            y += current[7];
-            break;
-
-          case 'A':
-            // TODO: optimize this
-            drawArc(ctx, x + l, y + t, [
-              current[1],
-              current[2],
-              current[3],
-              current[4],
-              current[5],
-              current[6] + l,
-              current[7] + t
-            ]);
-            x = current[6];
-            y = current[7];
-            break;
```

Comment:
@melchiar i wrote code in the file util/path.js to handle 'a' and 'A' at parse time, and also all the lower case commands.
I would like to get rid in this function also of case 'T' and case 'S'. and keep 'C' and 'Q'.
An 'S' would become a paricular different 'C' and a 'T' a particular different 'Q'.

The drawing code in this function explains how. and the code should be added in util/path.js function makePathSimpler. The transtion from s to S and to t to T is already handled.